packages/forms-web-app/src/dynamic-forms/journey-factory.js

```diff
@@ -1,8 +1,8 @@
 const { JOURNEY_TYPES } = require('@pins/common/src/dynamic-forms/journey-types');
-const { buildJourneyParams: buildHASLPAJourneyParams } = require('./has-questionnaire/journey');
```

Comment:
most of the code in here ends up redundant, it was just easier for my brain to port it so I could keep track of what I was changing

Reply:
removed all the redundant bits now